include/uart.h

```diff
@@ -99,6 +99,11 @@ enum uart_event_type {
 	 * - When RX timeout occurred, and data was stored in provided buffer.
 	 *   This can happen multiple times in the same buffer.
 	 * - When provided buffer is full.
+	 *
+	 * The data reflected in the event are stored in
```

Comment:
The word "reflected" is very confusing here, what is it supposed to mean? And the data is "is", not "are".

Reply:
Changed.  But I feel obligated to defend my usage, since it's been drilled into me so many times :smiley:  "data" is the plural form of "datum" (e.g. a single byte), but that usage is less common outside if scientific fields.

Reply:
> But I feel obligated to defend my usage, since it's been drilled into me so many times smiley "data" is the plural form of "datum"

Yeah, I agree, but modern usage in many fields is that "data" is uncountable noun, which semantically plural, but grammatically singular, similar to other words like "furniture", etc.

Thanks for the changes, looks clearer now.